src/vm/finalizerthread.cpp

```diff
@@ -539,7 +551,17 @@ void FinalizerThread::WaitForFinalizerEvent (CLREvent *event)
                 // Spawn thread to perform the profiler attach, then resume our wait
                 ProfilingAPIAttachDetach::ProcessSignaledAttachEvent();
                 break;
-#endif // FEATURE_PROFAPI_ATTACH_DETACH 
+#endif // FEATURE_PROFAPI_ATTACH_DETACH
+#ifdef FEATURE_PAL
+            case (WAIT_TIMEOUT + kLowMemoryNotification):
```

Comment:
The WaitForMultipleObjectsEx returns just WAIT_TIMEOUT, not WAIT_TIMEOUT + index